### PR TITLE
remove -v flag in drive-examples

### PR DESCRIPTION
**Pull request by cyanglaz**
The `-v` flag seems to cause a time-out in `flutter pub get`

### CODE REVIEW

**Review by dnfield — APPROVED** [review]
sa dLGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v.0.029
```
Comment (amirh) [inline CHANGELOG.md:1]: wrong version

**Review by amirh — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (cyanglaz) [reply CHANGELOG.md]: oops

**Comment by cyanglaz** [comment]
CI is queued for a while now, should we just land it? The change is pretty safe.

**Comment by cyanglaz** [comment]
Landing this to fix CI and try to unblock flutter/plugin

**Pull request merged**